src/conspiracies/corpusprocessing/umap_hdb.py

```diff
@@ -503,6 +522,14 @@ def main(
         help="Event to cluster. Must include name of source folder (newspapers or "
         "twitter) and event",
     )
+    parser.add_argument(
```

Comment:
You might be interested in radicli (almost no dependencies, but make creating cli's quite easy)

Reply:
This CLI is a remnant from Stine's branch. With the generic pipeline, this is not intended to be run on its own.

I'll consider it, but only if it truly makes my life easier with these extra third-party libraries.

Reply:
Only if it makes it easier